src/mma/GEMM.cpp

```diff
@@ -109,6 +109,12 @@ void GEMM::Impl::check_support() {
       throw std::runtime_error("Float32 input is not supported on Volta");
     }
   }
+
+  if (kernel_.GetPrecision().input_type == ValueType::float8e4m3) {
+    if (!hasFP8(device_)) {
+      throw std::runtime_error("FP8 input is not supported on this device");
```

Comment:
For consistency:
```suggestion
      throw std::runtime_error("Float8 input is not supported on this device");
```
Since we have `Float32` above.